documentation/database-documentation/database-schema.md

```diff
@@ -0,0 +1,12 @@
+#Interfaith Donation Database Schema
```

Comment:
Add a space between the Markdown header and word (#Interfaith Donation Database Schema should be # Interfaith Donation Database Schema). This allows GitHub to properly show Markdown files.